CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Changed
+- Upgrade to Gulp 4
+
+
```

Comment:
Remove empty line.

Reply:
How could I miss that 😱😄
Done in b7fa42b4ff51c8bfce89ccaad1bec0f6fd72e1dd